crates/bindings-csharp/BSATN.Runtime.Tests/Tests.cs

```diff
@@ -0,0 +1,179 @@
+namespace SpacetimeDB;
+
+using CsCheck;
+using Xunit;
+
+public static class BSATNRuntimeTests
+{
+    [Fact]
+    public static void AddressRoundtrips()
+    {
+        var str = "00112233445566778899AABBCCDDEEFF";
+        var addr = Address.FromHexString(str);
+
+        Assert.NotNull(addr);
+        Assert.Equal(addr.ToString(), str);
+
+        byte[] bytes =
```

Comment:
If you want, you can use [`Convert.FromHexString`](https://learn.microsoft.com/en-us/dotnet/api/system.convert.fromhexstring?view=net-8.0) here.

We can't use it in BSATN.Runtime because it's not part of .NET Standard which we need for Unity, but no harm in using it in tests.

Reply:
I see you used it in the identity roundtrips tests below, but not in this function. Seems unintentional?